src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ComboBoxTests/UnoSamples_Tests.ComboBoxTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Uno.UITest.Helpers;
+using Uno.UITest.Helpers.Queries;
+
+namespace SamplesApp.UITests.Windows_UI_Xaml_Controls.ComboBoxTests
+{
+	[TestFixture]
+	public partial class ComboBoxTests_Tests : SampleControlUITestBase
+	{
+		[Test]
+		public void ComboBoxTests_DefaultValue()
+		{
+			Run("SamplesApp.Wasm.Windows_UI_Xaml_Controls.ComboBox.ComboBox_ItemsSource");
```

Comment:
How can this test be testing the change you made? This sample isn't using the `Picker`.